sbin/attest-enroll

```diff
@@ -478,62 +478,48 @@ trap cleanup EXIT
 
 # We might need a TPM for what should be software-only things...
 start_swtpm() {
-	local -i tries port start ready
-	((port= 10240 + ( ($$ % (32768 - 10240) ) ) ))
+	local -i tries
+	local mypath
 	mkdir "${tmp}/swtpm"
-
-	start=$SECONDS
-	for ((tries=0, ready=0;
-	      ready == 0 && tries < 3 && (SECONDS - start) < 10;
-	      tries++, port+=2)); do
-
-		# swtpm --daemon exits immediately, which can be too soon.
-		swtpm socket							\
-			--tpm2							\
-			--tpmstate dir="${tmp}/swtpm"				\
-			--pid file="${tmp}/.pid"				\
-			--server type="tcp,bindaddr=127.0.0.1,port=$port"	\
-			--ctrl type="tcp,bindaddr=127.0.0.1,port=$((port+1))"	\
-			--flags startup-clear					&
-		swtpm_pid=$!
-
-		# Wait until swtpm is started and running
-		while ! lsof				\
+	tries=0
+	mypath=${tmp}/socket
+
+	# swtpm --daemon exits immediately, which can be too soon.
```

Comment:
`swtpm --daemon` should be fixed in new releases.  Did you also want to update the submodules?